manifests/base/runner-manager/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-manager
+  labels: {}
+spec:
+  selector: {}
+  template:
+    metadata: {}
+    spec:
+      serviceAccountName: default
+      containers:
+        - args:
+            - 'srcs.web.app:app'
+            - '--host'
+            - '--port'
+            - '80'
+          command:
+            - uvicorn
+          env:
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: redis-password
+                  name: redis
```

Comment:
Update according to above suggestion. Check also as they now belong to different folders if they are able to work with each other.

Reply:
ok